sql/catalyst/src/main/scala/org/apache/spark/sql/internal/connector/SupportsPushDownCatalystFilters.scala

```diff
@@ -35,7 +35,7 @@ trait SupportsPushDownCatalystFilters {
 
   /**
    * Returns the data filters that are pushed to the data source via
-   * {@link #pushFilters(Expression[])}.
+   * {@link #pushFilters(Predicate[])}.
```

Comment:
This change was wrong, wasn't it? `pushFilters` still takes a `Seq[Expression]`, not a `Predicate[]`. Seems it was wrong even before, since the parameter is a `Seq`, not an array.